docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -0,0 +1,293 @@
+          "properties": {
+            "name": {
+              "$id": "#/properties/info/properties/author/properties/name",
+              "type": "string",
+              "description": "Author's name."
+            },
+            "url": {
+              "$id": "#/properties/info/properties/author/properties/url",
+              "type": "string",
+              "description": "Link to author's website."
+            }
+          }
+        },
+        "description": {
+          "$id": "#/properties/info/properties/description",
+          "type": "string",
+          "description": "Description of plugin. Used on the plugins page in Grafana and for search on grafana.com."
+        },
+        "keywords": {
+          "$id": "#/properties/info/properties/keywords",
+          "type": "array",
+          "description": "Array of plugin keywords. Used for search on grafana.com.",
+          "items": {
+            "anyOf": [],
+            "$id": "#/properties/info/properties/keywords/items"
+          }
+        },
+        "links": {
+          "$id": "#/properties/info/properties/links",
+          "type": "array",
+          "description": "An array of link objects to be displayed on this plugin's project page in the form `{name: 'foo', url: 'http://example.com'}`",
+          "items": {
+            "anyOf": [],
+            "$id": "#/properties/info/properties/links/items"
+          }
+        },
+        "logos": {
+          "$id": "#/properties/info/properties/logos",
+          "type": "object",
+          "description": "SVG images that are used as plugin icons.",
+          "required": ["small", "large"],
+          "properties": {
+            "small": {
+              "$id": "#/properties/info/properties/logos/properties/small",
+              "type": "string",
+              "description": "Link to the \"small\" version of the plugin logo, which must be an SVG image. \"Large\" and \"small\" logos can be the same image.",
+              "examples": ["img/logo.svg"]
+            },
+            "large": {
+              "$id": "#/properties/info/properties/logos/properties/large",
+              "type": "string",
+              "description": "Link to the \"large\" version of the plugin logo, which must be an SVG image. \"Large\" and \"small\" logos can be the same image.",
+              "examples": ["img/logo.svg"]
+            }
+          }
+        },
+        "screenshots": {
+          "$id": "#/properties/info/properties/screenshots",
+          "type": "array",
+          "description": "An array of screenshot objects in the form `{name: 'bar', path: 'img/screenshot.png'}`",
+          "items": {
+            "$id": "#/properties/info/properties/screenshots/items"
+          }
+        },
+        "updated": {
+          "$id": "#/properties/info/properties/updated",
+          "type": "string",
+          "description": "Date when this plugin was built. Use `%TODAY%` for Grafana to autopopulate this value."
```

Comment:
We should a link or explain how %TODAY% and %VERSION% works. This only works if you are using the right tooling.

Reply:
Think we shouldn't reference %TODAY% and %VERSION% here. Include example of the actual values instead. Possibly link (in the future) to some documentation explaining usage of toolkit and %TODAY% and %VERSION%

Reply:
Removed them from the description, but kept them in the validation rule.